/* ==========================================================
 *
 * Generated by system . DO NOT EDIT!
 * ========================================================== */


// gsk_pb_req_data;
//----------------------------------------------------
message pb_req_t_req
{
    optional int32 f_cmd = 1;
    repeated int32  f_i_args = 2 ;
    repeated string f_s_args = 3;
}

/* ==========================================================
 *
 * Generated by system . DO NOT EDIT!
 * ========================================================== */


